Backpack/src/main/java/net/skyscanner/backpack/calendar/presenter/BpkCalendarController.kt

```diff
@@ -82,13 +76,10 @@ abstract class BpkCalendarController(
     onRangeSelected(selectedRange)
   }
 
-  // TODO: This will not respect differences in the order of the fields for each country.
-  // E.g. US = mm/dd/yyyy and UK = dd/mm/yyyy
-  // Shall we use DateFormat.getDateInstance instead?
-  internal fun getLocalizedDate(date: Date, pattern: String): String = SimpleDateFormat(pattern, locale).format(date)
+  internal fun getLocalizedDate(date: LocalDate, pattern: String): String = DateTimeFormatter.ofPattern(pattern, locale).format(date)
 
   open fun isToday(year: Int, month: Int, day: Int): Boolean {
-    return CalendarDay(year, month, day).date == CalendarDay.today().date
+    return LocalDate.of(year, month, day) == LocalDate.now()
   }
 
   fun updateSelection(selection: CalendarSelection) {
```

Comment:
Not sure why this is happening only now, but `selectedRange.end` should be null here too when the selection is a `SelectionDay`. This is causing the snapshots to fail.

Reply:
I can't seem to reproduce this. I don't think it's related to this PR